config/recipes/elastic-agent/fleet-kubernetes-integration-noroot.yaml

```diff
@@ -0,0 +1,295 @@
+---
```

Comment:
Maybe mention again in a header that this example only works from ECK 2.10.0

Reply:
Unfortunately I can't here, as we decode this in our e2e tests:
```
panic: failed to decode YAML: Object 'Kind' is missing in '# The following example manifest is only functional using ECK 2.10.0+.
' [recovered]
	panic: failed to decode YAML: Object 'Kind' is missing in '# The following example manifest is only functional using ECK 2.10.0+.
'
```
I've instead added this note to both the readme, and the docs which point to this file.